src/cf-forms/src/cf-forms.less

```diff
@@ -212,16 +112,16 @@ textarea {
     font-size: unit(@font-size / @base-font-size-px, em );
 }
 
-.error + .@{cf-forms_input-icon-class} {
-    color: @input-error;
+.a-text-input__error + .@{cf-forms_input-icon-class} {
```

Comment:
Icons felt out of scope, so they're left here but the correct input classes have been updated.